doc/whats_new/v0.21.rst

```diff
@@ -340,6 +340,16 @@ Support for Python 3.4 and below has been officially dropped.
   deterministic when trained in a multi-class setting on several threads.
   :issue:`13422` by :user:`Clément Doumouro <ClemDoum>`.
 
+- |Fix| Fixed bug in :func:`linear_model.ridge.ridge_regression` that
```

Comment:
This is also a bugfix for `RidgeClassifier` at least

Reply:
True, I updated the entry